### DISCUSSION TITLE
신고 누적된 게시글/댓글 삭제 관련

### DISCUSSION THREAD

**Discussion by hodin030** [cca-ffodregamdi/running-hi-back#169] [Trouble Shooting]
[기존] 
: 신고 누적 횟수 5회 이상인 경우 게시글/댓글 삭제, 
  관리자가 신고 거절 시 신고 데이터 삭제

[변경 사항]
: 관리자가 신고 수락할 시 게시글/댓글 상태값 변경(삭제하지 않음) -> 상태값 true인 게시글/댓글 조회 안 되도록 변경.
  관리자가 신고 거절 시에도 신고 데이터 유지(유저 항의 들어온 경우 대비)

**Comment by fakerdeft** [comment]
오 그렇군요~

**Comment by silverpoodle** [comment]
false -> true 로 바꿔야 할 것 같아유!!!!

**Reply by hodin030** [reply]
수정했습니다!

**Comment by hodin030** [comment]
근데 다시 생각해보니까 관리자가 신고 한번만 수락해도 상태값 true로 변경해야할 것 같아요

**Reply by fakerdeft** [reply]
정말요??

**Reply by hodin030** [reply]
왜냐면 관리자가 신고 수락 = 게시물에 문제 있음
이미 문제 있는 게시물을 신고 5번 들어올 때까지 그대로 두는 건데 그럼 안되지 않을까요?
만약 신고가 5번 안 들어오면 제제가 안되잖아요

**Reply by fakerdeft** [reply]
댓글 신고 상태 자체가 true로 되긴 해야 되네요

**Comment by Subak-Uncle** [comment]
확실히 한 번의 제제로 관리하는 것이 효율적이겠네요!!
